Makefile

```diff
@@ -145,6 +145,8 @@ standalone: node_modules
 .PHONY: add-on
 add-on: node_modules
 	mkdir -p $(BUILT)/$@
+	sed -n -e '/^%/p' $@/jar.mn | sed -e "s/^% //g" | sed -e "s/%/chrome\//g" > $@/chrome.manifest
```

Comment:
Can you separate this out to a separate rule please? No point in regenerating it each time - see some of the vendor library handling for instance.

Also, this should output direct to `$(BUILT)/add-on/chrome.manifest`, and we should git rm the existing`add-on/chrome.manifest`.